dojo/1.11/dojo.d.ts

```diff
@@ -784,8 +784,8 @@ declare namespace dojo {
 	/* dojo/Evented */
 
 	interface Evented {
-		on(type: string | ExtensionEvent, listener: EventListener): Handle;
-		emit(type: string | ExtensionEvent, event: any): boolean;
+		on(type: string | ExtensionEvent, listener: EventListener | Function): Handle;
+		emit(type: string | ExtensionEvent, ...events: any[]): boolean;
```

Comment:
Right, I understand that part, was just confused for a moment by the any[], but yeah, that makes sense.